src/components/Header/Header.jsx

```diff
@@ -47,7 +48,7 @@ const LinkGroup = () => (
     {navbarPages.map(page => (
       <div key={page.name} className={styles.tabContainer}>
         {page.external ?
-          <a href={page.path} className={styles.tab}>{page.name}</a> :
+          <a href={page.path} className={styles.tab} target="_blank">{page.name}</a> :
```

Comment:
you need to do this in BurgerMenu component as well (or refactor it out somewhere since the code is duplicated)